gneiss/plot/_decompose.py

```diff
@@ -155,3 +155,137 @@ def balance_barplots(tree, balance_name, header, feature_metadata,
     ax_num.set_xlim([0,  max([num_.max().values[1],
                               denom_.max().values[1]])])
     return ax_num, ax_denom
+
+
+def proportion_plot(table, metadata, category, left_group, right_group,
+                    num_features, denom_features,
+                    feature_metadata=None,
+                    label_col='species',
+                    num_color='#105d33', denom_color='#b0d78e',
+                    axes=(None, None)):
+    """ Plot the mean proportions of features within a balance.
+
+    This plots the numerator and denominator components within a balance.
+
+    Parameters
+    ----------
+    table : pd.DataFrame
```

Comment:
are you concerned about requiring dense matrices for the table data?

Reply:
Nope